src/types.tsx

```diff
@@ -493,8 +493,15 @@ export interface ScreenStackProps extends ViewProps {
    * A callback that gets called when the current screen finishes its transition.
    */
   onFinishTransitioning?: (e: NativeSyntheticEvent<TargetedEvent>) => void;
-  gestureDetectorBridge?: React.MutableRefObject<GestureDetectorBridge>;
   ref?: React.MutableRefObject<React.Ref<View>>;
+  /**
+   * Props for custom screen transition (ScreenGestureDetector)
+   */
+  screensRefs?: GestureProviderProps['screensRefs'];
+  currentScreenId?: GestureProviderProps['currentScreenId'];
+  goBackGesture?: GestureProviderProps['goBackGesture'];
+  screenEdgeGesture?: GestureProviderProps['screenEdgeGesture'];
+  transitionAnimation?: GestureProviderProps['transitionAnimation'];
```

Comment:
Can we make something with these being repeated here? `keyof ...` operator?

Reply:
What about changing into interfaces? Changed in: https://github.com/software-mansion/react-native-screens/pull/2449/commits/79571e2f9317eedfe272ca7ece1d3589f443dcad